HHS/Head-Start-TTADP
# [TTAHUB-1121] Fix other-entity duplicating objectives and not showing TTA provided on visit

## Description of change

A bug was discovered if you create a OE report with multiple entities.

## How to test

1. Create an OE report with multiple entities
2. Add objectives with tta
2. Save and leave the report
3. Revist the report notice there is one objective per entity and TTA is blank

We should test creating an OE with multiple entities taking it all the way to approval. Make sure the Review and Approval screens show correctly.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1121


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
